### PR TITLE
Use httpstatus module

### PR DESCRIPTION
**Pull request by mshriver**
Use the module imports instead of magic numbers. They probably won't change but some linting/formatting/mypy rules start to enforce it.

These were all the uses of the status codes that I found throughout the backend codebase. There may be some others that are hiding but it's not critical to get 100% of them in this PR.

### CODE REVIEW

File: backend/ibutsu_server/__init__.py [hunk 1]
```diff
@@ -137,31 +138,31 @@ def run_task():
         # get params
         params = request.get_json(force=True, silent=True)
         if not params:
-            return "Bad request", 400
+            return HTTPStatus.BAD_REQUEST.phrase, HTTPStatus.BAD_REQUEST
         # get user info
         token = params.get("token")
         if not token:
-            return "Unauthorized", 401
+            return ("Unauthorized",)
```
Comment (mshriver) [inline backend/ibutsu_server/__init__.py:145]: Fixing

**Review by digitronik — APPROVED** [review]
LGTM

File: backend/ibutsu_server/controllers/import_controller.py [hunk 2]
```diff
@@ -89,4 +90,4 @@ def add_import(
         run_archive_import.delay(new_import.to_dict())
     else:
         return "Unsupported Media Type", 415
-    return new_import.to_dict(), 202
+    return new_import.to_dict(), 2
```
Comment (Griffin-Sullivan) [inline backend/ibutsu_server/controllers/import_controller.py:93]:
```suggestion
    return new_import.to_dict(), HTTPStatus.ACCEPTED
```
If we change this line shouldn't it be with HTTPStatus?

**Review by Griffin-Sullivan — APPROVED** [review]

File: backend/ibutsu_server/controllers/import_controller.py (hunk 2 above)
Comment (mshriver) [reply backend/ibutsu_server/controllers/import_controller.py:93]: great eye and catch! This is definitely a typo or bad find+replace entry.

**Pull request merged**